website/docs/reference/projects.md

```diff
@@ -43,13 +43,15 @@ The UI shows the currently available projects. To create a new project, use the
 
 The configuration of a new Project is now available. the following input is available to create the new Project.
 
-![A project creation form. The form fields are labeled "project ID", "name", and "description". The "Create" button is highlighted.](/img/projects_save_new_project.png)
-
-| Item         | Description                        |
-| ------------ | ---------------------------------- |
-| Project Id   | Id for this Project                |
-| Project name | The name of the Project.           |
-| Description  | A short description of the project |
+![A project creation form. The form fields are labeled "project ID", "name", "description", "mode" and "defaultStickiness". The "Create" button is highlighted.](/img/projects_save_new_project_v2.png)
```

Comment:
Non-blocking: This is my fault, but I think that description is more wordy than what it has to be. Something like this, maybe?

```suggestion
![A project creation form. The "Create" button is highlighted.](/img/projects_save_new_project_v2.png)
```